smoke/README.md

```diff
@@ -0,0 +1,19 @@
+# QA for OpenShift Shared Resource CSI Driver
+<pre>
+This is a Behavior-driven development (or BDD) test framework leveraging `behave` for `python`. BDD is an agile software development technique that encourages collaboration between developers, QA and non-technical or business participants in a software project.Behave uses tests written in a natural language style, backed up by Python code.
+
+We are using this framework to achieve the QA verification of any Jira Stories development takes on for the OpenShift Shared Resource CSI Driver, as well as any regression test suites the team want to cultivate for the OpenShift Shared Resource CSI Driver.
+
+Hence, the reader should consider these tests the 'next phase' of testing after the [golang based tests](https://github.com/openshift/csi-driver-shared-resource/tree/master/test/e2e) that run in OpenShift CI prow jobs as part of merging pull requests with code changes.
+</pre>
+
```

Comment:
You need to supply a workflow on *HOW* a new smoke test is added 
- add the feature file in a certain dir
- add the py file in a certain dir
- any naming conventions
- any refs to `behave` docs that might help new writers 
- refs and brief explanations for all the nice python utilities you've added for interacting with openshift like openshift.py., projects.py, steps.py, etc.

Similar to what @otaviof did for the BATS based e2e framework for shipwright CLI, like at https://github.com/shipwright-io/cli/blob/main/docs/testing.md#bats and https://github.com/shipwright-io/cli/blob/main/docs/testing.md#test-cases

Reply:
And make sure you also cover all of my questions from https://github.com/openshift/csi-driver-shared-resource/pull/62#discussion_r758516919 as well wrt describing the process and methodology here moving forward

Reply:
I have addressed the concerns and updated the README file